index.css

```diff
@@ -37,7 +37,7 @@ body {
   align-content: center;
   justify-content: center;
 
-  overflow-y: auto;
```

Comment:
y tho? do you even use Y scrolling? for gldemo at least, the `height:100%` regime kept showing a futile scrollbar that did nothing...